Packs/McAfee_ESM_v2/Integrations/McAfee_ESM_v2/McAfee_ESM_v2.yml

```diff
@@ -0,0 +1,738 @@
+      - CURRENT_MONTH
+      - PREVIOUS_WEEK
+      - CURRENT_WEEK
+      required: false
+      secret: false
+    - default: false
+      description: 'If the timeRange argument is set to CUSTOM, the start time for the time range. For example: 2017-06-01T12:48:16.734Z'
+      isArray: false
+      name: customStart
+      required: false
+      secret: false
+    - default: false
+      description: 'If the timeRange argument is set to CUSTOM, the end time for the time range. For example: 2017-06-01T12:48:16.734Z'
+      isArray: false
+      name: customEnd
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'User assigned to handle the triggered alarm. Use the ''ME'' option
+        to use the instance user, or use [format EsmUser](https://<esm-ip>:<esm-port>/rs/esm/help/types/EsmUser). '
+      isArray: false
+      name: assignedUser
+      predefined:
+      - ME
+      - ''
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieves a list of triggered alarms.
+    execution: false
+    name: esm-fetch-alarms
+    outputs:
+    - contextPath: McAfeeESM.Alarm.ID
+      description: Alarm ID.
+      type: number
+    - contextPath: McAfeeESM.Alarm.summary
+      description: Alarm summary.
+      type: string
+    - contextPath: McAfeeESM.Alarm.assignee
+      description: Alarm assignee.
+      type: string
+    - contextPath: McAfeeESM.Alarm.severity
+      description: Alarm severity.
+      type: number
+    - contextPath: McAfeeESM.Alarm.triggeredDate
+      description: Alarm triggered date.
+      type: date
+    - contextPath: McAfeeESM.Alarm.acknowledgedDate
+      description: Alarm acknowledged date.
+      type: date
+    - contextPath: McAfeeESM.Alarm.acknowledgedUsername
+      description: Alarm acknowledged username.
+      type: string
+    - contextPath: McAfeeESM.Alarm.alarmName
+      description: Alarm name.
+      type: string
+    - contextPath: McAfeeESM.Alarm.conditionType
+      description: Alarm condition type.
+      type: number
+  - arguments:
+    - default: false
+      description: 'Filters for cases that were opened before this date. In the format "<number><timeunit>", for example: 1 day,30 minutes,2 weeks,6 months,1 year'
+      isArray: false
+      name: since
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets a list of cases from McAfee ESM.
+    execution: false
+    name: esm-get-case-list
+    outputs:
+    - contextPath: McAfeeESM.Case.ID
+      description: Case ID.
+      type: number
+    - contextPath: McAfeeESM.Case.Summary
+      description: The summary of the case.
+      type: string
+    - contextPath: McAfeeESM.Case.Status
+      description: The status of the case.
+      type: string
+    - contextPath: McAfeeESM.Case.OpenTime
+      description: The date and time when the case was opened.
+      type: date
+    - contextPath: McAfeeESM.Case.Severity
```

Comment:
what about `McAfeeESM.Case.Assignee`, `McAfeeESM.Case.Organization`, `McAfeeESM.Case.EventList`, `McAfeeESM.Case.Notes`?
all appeared in the original integration

Reply:
It’s not a part of the data in this API call.